README.md

```diff
@@ -22,18 +22,22 @@ Explanations of all parameters can be seen by using the help flag, i.e. `cargo r
-- `halt_increase_threshold` (environment variable: `EUR2CCD_SERVICE_HALT_INCREASE_THRESHOLD`): Determines the threshold where an update increasing the exchange rate triggers a halt, specified in percentages.  (default: 100%)
-- `warning_decrease_threshold` (environment variable: `EUR2CCD_SERVICE_WARNING_DECREASE_THRESHOLD`): Determines the threshold where an update decreasing the exchange rate triggers a warning, specified in percentages. (default: 15%)
-- `halt_decrease_threshold` (environment variable: `EUR2CCD_SERVICE_HALT_DECREASE_THRESHOLD`): Determines the threshold where an update decreasing the exchange rate triggers a halt, specified in percentages.  (default: 50%)
-
+- `pull-interval` (environment variable: `EUR2CCD_SERVICE_PULL_INTERVAL`): How often to read the exchange rate from each source (In seconds). (default: 60 seconds)
+- `max-rates-saved` (environment variable: `EUR2CCD_SERVICE_MAX_RATES_SAVED`): How many exchange rates should be saved at a time from each source (and used to determine the update value). (default: 60) 
+- `update-interval` (environment variable: `EUR2CCD_SERVICE_UPDATE_INTERVAL`): How often to update the exchange rate on chain (In seconds). (default: 1800 seconds)
+- `warning-increase-threshold` (environment variable: `EUR2CCD_SERVICE_WARNING_INCREASE_THRESHOLD`): Determines the threshold where an update increasing the exchange rate triggers a warning, specified in percentages. (default: 30%)
+- `halt-increase-threshold` (environment variable: `EUR2CCD_SERVICE_HALT_INCREASE_THRESHOLD`): Determines the threshold where an update increasing the exchange rate triggers a halt, specified in percentages.  (default: 100%)
+- `warning-decrease-threshold` (environment variable: `EUR2CCD_SERVICE_WARNING_DECREASE_THRESHOLD`): Determines the threshold where an update decreasing the exchange rate triggers a warning, specified in percentages. (default: 15%)
+- `halt-decrease-threshold` (environment variable: `EUR2CCD_SERVICE_HALT_DECREASE_THRESHOLD`): Determines the threshold where an update decreasing the exchange rate triggers a halt, specified in percentages.  (default: 50%)
+- `coin-gecko` (environment variable:  `EUR2CCD_SERVICE_COIN_MARKET_CAP`): If this flag is set, the service will use Coin Gecko as a source.
+- `live-coin-watch` (environment variable:  `EUR2CCD_SERVICE_LIVE_COIN_WATCH`): If this flag is set, the service will use Live Coin Watch as a source. The value is expected to be an API key for the site.
+- `coin-market-cap` (environment variable:  `EUR2CCD_SERVICE_COIN_MARKET_CAP`): If this flag is set, the service will use Coin Market Cap as a source. The value is expected to be an API key for the site.
+- `bitfinex` (environment variable:  `EUR2CCD_SERVICE_BITFINEX`): If this flag is set, the service will use Bitfinex as a source.
```

Comment:
I've left Bitfinex as a potential source, because I've been using it while testing, but it should probably be removed.